src/docker-osx-dev

```diff
@@ -772,6 +774,18 @@ function watch {
   done
 }
 
+#
+# Usage: prepare_sync
+#
+# Configure your development paths prior to sync (or watching) to ensure the
+# appropriate files are excluded and/or included.
+#
+function prepare_sync {
+  configure_paths_to_sync "$docker_compose_file" "${paths_to_sync[@]}"
```

Comment:
I am not strong enough in BASH to know the answer; but all accounts it looks like it *shouldn't* work, so my testing is probably too naive.  I'll put back the code without the convenience function